script.js

```diff
@@ -1,4 +1,116 @@
+//-------------------------------------------Base Code---------------------------------------
+//array to store final number concatenated by the players
+var diceRolls = [];
+//random number rolled by dice 1
+var dice1 = 0;
+//random number rolled by dice 2
+var dice2 = 0;
+//number of turns completed. Each round there will be 2 turns since there are only 2 players
+var playerCount = 0;
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+  while (playerCount<2){
+    //generate dice roll numbers
+    if (input == ''){
+      dice1=Math.floor(Math.random()*6+1);
+      dice2=Math.floor(Math.random()*6+1);
+      console.log(playerCount);
+      return `Your dice numbers are ${dice1} and ${dice2}. To put ${dice1} as your first number, enter "Dice 1". Otherwise, enter "Dice 2" to put ${dice2} as your first number`
+    }
+    //add final number concatenated by players to an array to be stored so the numbers can compared at the end of the game to see who won
+    if (input =='Dice 1'){
+      playerNumber=`${dice1}${dice2}`
+      diceRolls.push(Number(playerNumber));
+      playerCount = playerCount + 1
+      return `Your number is ${playerNumber}`
```

Comment:
Lines 22-24 and lines 29-31 are repeated. You could have put them outside of the if statements to avoid repeating yourself. Please don't repeat code when you can avoid it, because repeated code increases chances of bugs.